README.md

```diff
@@ -4,28 +4,25 @@
-already trained state of the art model for performing various flavour of separation :
+**Spleeter** is a source separation library written in [Python](https://www.python.org/) and utilizing [Tensorflow](https://tensorflow.org/), created by [Deezer](https://www.deezer.com/). It makes it easy to train source separation models (assuming you have a dataset of isolated sources), and provides already-trained state-of-the-art models for performing various flavours of separation:
 
 * Vocals (singing voice) / accompaniment separation ([2 stems](https://github.com/deezer/spleeter/wiki/2.-Getting-started#using-2stems-model))
 * Vocals / drums / bass / other separation ([4 stems](https://github.com/deezer/spleeter/wiki/2.-Getting-started#using-4stems-model))
 * Vocals / drums / bass / piano / other separation ([5 stems](https://github.com/deezer/spleeter/wiki/2.-Getting-started#using-5stems-model))
 
-2 stems and 4 stems models have [high performances](https://github.com/deezer/spleeter/wiki/Separation-Performances) on the [musdb](https://sigsep.github.io/datasets/musdb.html) dataset. **Spleeter** is also very fast as it can perform separation of audio files to 4 stems 100x faster than real-time when run on a GPU.
+2 stem and 4 stem models have [high performances](https://github.com/deezer/spleeter/wiki/Separation-Performances) on the [musdb](https://sigsep.github.io/datasets/musdb.html) dataset. **Spleeter** is also very fast - it can perform separation of audio files to 4 stems **100x faster than real-time** when run on a GPU.
 
-We designed **Spleeter** so you can use it straight from [command line](https://github.com/deezer/spleeter/wiki/2.-Getting-started#usage)
+We designed **Spleeter** so you can use it straight from the [command line](https://github.com/deezer/spleeter/wiki/2.-Getting-started#usage),
 as well as directly in your own development pipeline as a [Python library](https://github.com/deezer/spleeter/wiki/4.-API-Reference#separator). It can be installed with [Conda](https://github.com/deezer/spleeter/wiki/1.-Installation#using-conda),
-with [pip](https://github.com/deezer/spleeter/wiki/1.-Installation#using-pip) or be used with
+with [pip](https://github.com/deezer/spleeter/wiki/1.-Installation#using-pip) or used with
 [Docker](https://github.com/deezer/spleeter/wiki/2.-Getting-started#using-docker-image).
 
 
 ## Quick start
 
-Want to try it out but don't want to install anything ? We have setup a [Google Colab](https://colab.research.google.com/github/deezer/spleeter/blob/master/spleeter.ipynb).
+Want to try out Spleeter but don't want to install anything? We have a [Google Colab](https://colab.research.google.com/github/deezer/spleeter/blob/master/spleeter.ipynb) set up. Simply click the play button next to each command to run it on the Google Colab system. This will allow you to try separating the vocals and accompaniment stems from an example audio file.
```

Comment:
Hi @ttepatti 

Thanks for the PR. I was going through your changes and not sure why you'd need to add this?

Reply:
Probably caused by hacktoberfest.

Reply:
Hey there! I added that part in addition to the grammar fixes because I personally was unfamiliar with Google Colab, and thought others might be as well. Its easy to scroll past if you're a beginner, but funnily enough, it's one of the most useful tools for a beginner to try playing with Spleeter (since they don't have to go through the installation process on their own machine). I figured a small introductory sentence or two could be helpful.